11주차/(P)60058/60058_python_김규리.py

```diff
@@ -0,0 +1,47 @@
+def solution(p):
+    length = len(p)
+    countOpen = 0
+    countClose = 0
+    balance = 0
+    
+    u=""
+    v=""
+    
+    if length == 0:
+        return p
+    elif length >= 2:
+        for i, c in enumerate(p):
+            if c=="(":
+                countOpen +=1
+            elif c==")":
+                countClose +=1
+            if countOpen==countClose:
+                u = p[:i+1]
+                v = p[i+1:length]
+                print("u: " + u + " v: " + v)
+                break;
+                
+        for c in u:
+            if c =="(":
+                balance+=1
+            elif c==")":
+                balance-=1
+            if balance < 0:
+                str = "("
+                v = solution(v)
+                str += v
+                str += ")"
+                str += reverse(u)
+                return str
+        v = solution(v)
+        return u+v
```

Comment:
마찬가지로 v는 solution function에서만 사용되기 때문에 굳이 할당하지 않아도 될 것 같네요
```suggestion
        return u+solution(v)
```